client/app.js

```diff
@@ -42,7 +42,6 @@ const porybox = ng.module('porybox', [
 ]);
 
 porybox.controller('MainCtrl', function () {
-  this.boxes = [];
```

Comment:
True, but it won't be undefined because [we already initialize it to an empty array if it's undefined](https://github.com/porygonco/porybox/blob/2242535dcaa00299d475b347c94d2271aea1a07e/client/layout.ejs#L40).